tests/unit/aiplatform/test_pipeline_jobs.py

```diff
@@ -621,6 +622,99 @@ def test_run_call_pipeline_service_create_with_timeout_not_explicitly_set(
             timeout=None,
         )
 
+    @pytest.mark.parametrize(
+        "job_spec",
+        [_TEST_PIPELINE_SPEC_JSON, _TEST_PIPELINE_SPEC_YAML, _TEST_PIPELINE_JOB],
+    )
+    @pytest.mark.parametrize(
+        "failure_policy",
+        [
+            (
+                "slow",
+                pipeline_failure_policy.PipelineFailurePolicy.PIPELINE_FAILURE_POLICY_FAIL_SLOW,
+            ),
+            (
+                "fast",
+                pipeline_failure_policy.PipelineFailurePolicy.PIPELINE_FAILURE_POLICY_FAIL_FAST,
+            ),
+        ],
+    )
+    @pytest.mark.parametrize("sync", [True, False])
+    def test_run_call_pipeline_service_create_with_failure_policy(
```

Comment:
Is there a way to test the result of setting `failure_policy` to fast vs. slow in a completed PipelineJob, or do you think we should leave this type of test to the service team?

Reply:
I think we should leave the test to the service team.. They have tests on the backend that would make sure that it is working as intended.